packages/terra-application-navigation/CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ Unreleased
 ### Fixed
 * Fix wdio test url.
 
+### Changed
+* Duplicate ID in example changed.
```

Comment:
This should be moved to the unreleased section.

Reply:
Resolved in 8f95e89